ios/sdk/src/dropbox/Dropbox.m

```diff
@@ -128,25 +131,31 @@ + (BOOL)application:(UIApplication *)app
     if (currentReject == nil || currentResolve == nil) {
         return NO;
     }
-    DBOAuthResult *authResult = [DBClientsManager handleRedirectURL:url];
-    if (authResult) {
-        if ([authResult isSuccess]) {
-            currentResolve(authResult.accessToken.accessToken);
-        } else {
-            NSString *msg;
-            if ([authResult isError]) {
-                msg = [NSString stringWithFormat:@"%@, error type: %zd",[authResult errorDescription], [authResult errorType]];
+    
+    BOOL canHandle = [DBClientsManager handleRedirectURL:url completion:^(DBOAuthResult *authResult) {
+        if (authResult) {
```

Comment:
Is it possible to get nil here? What does it mean?

Reply:
I don't know. That's how the previous implementation was done and that's how it's done in the dropbox documentation example.

Reply:
Ack.